sympy/geometry/polygon.py

```diff
@@ -2282,6 +2282,32 @@ def nine_point_circle(self):
         """
         return Circle(*self.medial.vertices)
 
+    @property
+    def eulerline(self):
+        """The Euler line of the triangle.
+
+        The line which passes through circumcenter, centroid and orthocenter.
+
+        Returns
+        =======
+
+        eulerline : Line except equilateral triangle because in the equilateral
```

Comment:
avoid personal pronouns in docstrings:

->
eulerline : Line (or Point for equilateral triangles in which case all centers coincide)